src/main/java/emissary/grpc/pool/ConnectionFactory.java

```diff
@@ -285,11 +299,31 @@ public int getMaxInboundMetadataSize() {
         return maxInboundMetadataSize;
     }
 
-    public String getLoadBalancingPolicy() {
-        return loadBalancingPolicy;
-    }
-
     public float getErodingPoolFactor() {
         return erodingPoolFactor;
     }
+
+    public boolean getPoolBlockedWhenExhausted() {
+        return this.poolConfig.getBlockWhenExhausted();
+    }
+
+    public long getPoolMaxWaitMs() {
```

Comment:
Consider renaming to make the time unit clearer (e.g., `getPoolMaxWaitMillis`) otherwise it's not clear whether we're talking milliseconds or microseconds.